Add more Quarg news (#9371)
endless-sky/endless-sky: data/quarg/quarg news.txt

			"."
		word
			`"`

news "quarg anywhere"
	location

> 		government "Quarg"

Reviewer: Would this not trigger in regions where quarg don't speak human languages well, or at all?

Reviewer: You probably need to do regional filtering, using the `human quarg`, `hai quarg` attributes. (iirc those two speak human languages well)

Also, what about Humanika? I never know which filters account for it.

Author: Humanika has `farming human south` for its attributes.
Given the Quarg basically solely interact there via the robots, I don't think it should be gettin the news as an exception to the attributes.

	name
		word
			"Quarg"
	message
		word
			`"Yes human? Do you have any questions?"`
			`"Hello human. Are you enjoying your time on our world? We are glad to receive visitors."`
			`"I am sorry, but we will not provide you with our technology. It is for the best that humans grow and develop on their own."`
			`"It troubles me greatly to see how many species still cling to senseless fighting. I can only hope you will all come to see reason."`
			`"Last year our scientific efforts were quite fruitful, but I am afraid we cannot simply share our discoveries with you."`
			`"If you ever observe some piece of our technology in the possession of someone else, please report it to us. We cannot allow our equipment to be in the wrong hands."`
			`"You are welcome to be amongst us here, provided you seek no conflict with anyone on our worlds."`
			`There's an eerie silence as a particularly large crowd of Quarg walks by you.`

> 			`A group of Quarg is enthralled in what seems an engaging discussion, until one more Quarg comes to them. They all turn to it, after it says a quick sentences, they all get up in unison and move out to somewhere in the hangars.`

Reviewer 2: Maybe:
```suggestion
			`A group of Quarg is enthralled in what seems an engaging discussion, until one more Quarg comes to them. They all turn to it, and after it says something quickly they all get up in unison and move to somewhere else in the hangars.`
```

> 			`An older Quarg is playing with a much younger Quarg, young enough that it's still shorter than yourself. The strange game consists of passing to one another a jar with some slimy, metallic substance inside. Every time the jar is passed, the substance meshes itself into a different shape, sometimes into shapes you know not the meaning of.`

Reviewer 3: ```suggestion
			`An older Quarg is playing with a much younger Quarg, young enough that they're still shorter than yourself. The strange game consists of passing to one another a jar with some slimy, metallic substance inside. Every time the jar is passed, the substance meshes itself into a different shape, sometimes into shapes you know not the meaning of.`
```
This isnt my area of course, but still isnt it better this way? Or did we not get to the quarg with this issue yet? They're obviously sentient

Author: It's intentional that the Quarg pronouns are it/its, yes.

Reviewer 3: that's weird but ok then we shouldnt change it in this PR

			`Dozens of Quarg children are being led around by a handful of adults, like if in a school trip. When they notice you, they continue taking glances and talking among themselves. It seems they've never seen a human before.`

news "ringworld quarg"
	location
		government "Quarg"
		attributes "station"
	name
		word
			"Quarg"
	message
		phrase

> 			"ringworld quarg phrases" 5

Reviewer 4: ```suggestion
			"ringworld quarg phrases" 8
```
Feels like the poems should at least be rarer.

			"quarg poems on species" 1

phrase "ringworld quarg phrases"
	word
		`You see a shorter, younger Quarg, likely a child in Quarg years, being shown around the ringworld by an adult. From the looks of it the parent is teaching it about the ring.`

> 		`A Quarg is standing uncomfortably close to the walls, tapping it slowly with both palms. You observe it for a few minutes, until it stops abruptly, and goes about its business like the others.`

Reviewer 2: ```suggestion
		`A Quarg is standing uncomfortably close to the walls, tapping it slowly with both palms. You observe it for a few minutes, until it stops abruptly and goes about its business like the others.`
```

> 		`Standard activity in the ringworld is stopped by a few dozen Quarg in heavily armored suits, moving in a quick pace and escorting a large, tube-like container which floats to accompany them. Other Quarg are quick to make way for the group, steering a great distance from them.`

Reviewer 2: ```suggestion
		`Usual activity in the ringworld is stopped by a few dozen Quarg in heavily armored suits moving at a quick pace and escorting a large, tube-like container which floats to accompany them. Other Quarg are quick to make way for the group, allowing themselves a great distance from the armored Quarg.`
```

		`Some Quarg scientists are dragging along a whole train of linked lab carts, packed fully with utensils, chemicals, components and reagents. It takes nearly half an hour before you see the last of the carts pass by.`

> 		`Hundreds of Quarg are blocking most of one of the largest, widest corridors in the ring. They're all sitting down on the floor, staring down blankly in complete silence, with a scarce few seldom glancing up to the ceiling, only to move their gaze downwards once again. Quarg who witness the scene stop any and all conversation, moving elsewhere.`

Reviewer 2: ```suggestion
		`Hundreds of Quarg are blocking most of one of the largest, widest corridors in the ring. They're all sitting down on the floor, staring down blankly in complete silence, with a scarce few seldom glancing up to the ceiling only to move their gaze downwards once again. You notice that Quarg who witness the scene stop all conversation and move elsewhere.`
```

		`One Quarg walks up to you, and stares down into your eyes. It stays there in silence, just looking at you, before finally blinking once and moving along, disappearing into the ring's hallways.`

> 		`A Quarg is overseeing the unloading of dozens of ship-sized cargo containers that have arrived in the ring. It seems to mentally count the cargo as it's moved out in perfectly identical sets. After what must've been thousands of cargo tons, it clicks off one of dozens of small dots on a data pad, and begins counting again.`

Reviewer 2: ```suggestion
		`A Quarg is overseeing the unloading of dozens of ship-sized cargo containers that have arrived in the ring. It seems to mentally count the cargo as it's moved out in perfectly identical sets. After what must've been thousands of cargo tons, it clicks off one of dozens of small dots on a data pad and begins counting again.`
```

> 		`Two Quarg ships land at the same time a few hangar bays away from one another. They open their hatches, and what must be the ships' entire crews come out. The crew of one ship heads to the other, and vice-versa. The crews each board the other's ship, and depart.`

Reviewer 2: ```suggestion
		`Two Quarg ships land at the same time a few hangar bays away from one another. They open their hatches, and what must be the ships' entire crews come out. Without a hitch the crew of each ship boards the other's ship and depart.`
```

		`The Quarg seem to have set up some form of festival in the ring: endless stands stretch out as far as you can see, all displaying variations of the same orange-like fruit, ranging from impossibly smooth to unbelievably wrinkled, and also from sizes as small as a pea to as large as a watermelon. You can't figure what quality they like best.`
		`You see one Quarg turn a sharp corner in what's a haste by their standards. When you look into the corridor it went to, you see it's a dead end, with the Quarg having inexplicably vanished.`

> phrase "quarg poems on species"

Reviewer 4: I don't know that much about the Quarg, but them having poetry seems a little strange.

	word
		"Resolute will / In ancient bone / Climb to claim / A godless throne"

> 		"Pulling strings / Stage nonstop strife / Clueless puppets / May yet survive"

Reviewer: Anti-Quarg messages?

~~Good.~~

			"."
		word
			`"`

news "quarg anywhere"
	location
		attributes "quarg" "human quarg" "hai quarg"
		not attributes "korath quarg" "gegno quarg"
	name
		word
			"Quarg"
	message
		word

> 			`"Yes human? Do you have any questions?"`
> 			`"Hello human. Are you enjoying your time on our world? We are glad to receive visitors."`

Reviewer 2: ```suggestion
			`"Yes, human? Do you have any questions?"`
			`"Hello, human. Are you enjoying your time on our world? We are glad to receive visitors."`
```
It seems to me that the Quarg would be a patient species and that it might be reflected in them pausing.

			`"Last year our scientific efforts were quite fruitful, but I am afraid we cannot simply share our discoveries with you."`
			`"If you ever observe some piece of our technology in the possession of someone else, please report it to us. We cannot allow our equipment to be in the wrong hands."`
			`"You are welcome to be amongst us here, provided you seek no conflict with anyone on our worlds."`
			`There's an eerie silence as a particularly large crowd of Quarg walks by you.`
			`A group of Quarg is enthralled in what seems an engaging discussion, until one more Quarg comes to them. They all turn to it, and after it says something quickly they all get up in unison and move to somewhere else in the hangars.`
			`An older Quarg is playing with a much younger Quarg, young enough that it's still shorter than yourself. The strange game consists of passing to one another a jar with some slimy, metallic substance inside. Every time the jar is passed, the substance meshes itself into a different shape, sometimes into shapes you know not the meaning of.`

> 			`Dozens of Quarg children are being led around by a handful of adults, like if on a school trip. When they notice you, they continue taking glances and talking among themselves. It seems they've never seen a human before.`

Reviewer 2: ```suggestion
			`Dozens of Quarg children are being led around by a handful of adults, as if on a school trip. When they notice you, they continue taking glances and talking among themselves. It seems they've never seen a human before.`
```
"As if" sounds better to me but "like if" still makes sense

news "ringworld quarg"
	location
		attributes "quarg"
		attributes "station"
		not planet "Lagrange"
	name
		word
			"Quarg"
	message
		phrase
			"ringworld quarg phrases" 8
			"quarg poems on species" 1

phrase "ringworld quarg phrases"
	word

> 		`You see a shorter, younger Quarg, likely a child in Quarg years, being shown around the ringworld by an adult. From the looks of it the parent is teaching it about the ring.`

Reviewer 2: "From the looks of it" feels ambiguous in meaning. Does it refer to the actions of the child, or the adult, or both? The new comma doesn't particularly help for me. Changing it to "It seems like" or "It looks like" might be better.

Repeating "it" is also potentially confusing, so maybe we should change the second "it" to "the child". Although, that one is probably not as big of a deal.
```suggestion
		`You see a shorter, younger Quarg, likely a child in Quarg years, being shown around the ringworld by an adult. It seems like the parent is teaching the child about the ring.`
```
